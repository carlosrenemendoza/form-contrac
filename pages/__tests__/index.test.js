import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Main from '../index';

describe('<Main />', () => {
  it('Should rendered correctly', () => {
    const component = shallow(<Main />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
