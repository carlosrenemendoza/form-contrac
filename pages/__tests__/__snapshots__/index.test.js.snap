// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should rendered correctly 1`] = `
<div
  className="App"
>
  <Header />
  <creditCard />
</div>
`;
